### DISCUSSION TITLE
Which US president nickname is after 🧸?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#827] [Galaxy Trivia] [answered]
.

**Comment by hasanerzi** [comment]
The US president whose nickname is associated with a bear is Theodore Roosevelt. He was often referred to as "Teddy" Roosevelt, and the name "Teddy" originated from an incident during a hunting trip in 1902. Roosevelt had spared the life of a bear cub, and the event inspired a political cartoon that depicted him with the bear. The cartoon gained popularity, and a toymaker named Morris Michtom created a stuffed bear called "Teddy's Bear" in honor of the president. The toy became a sensation, and the nickname "Teddy" stuck with Roosevelt.

**Comment by peterbaccaro** [comment, marked as answer]
The U.S. president who is nicknamed "Teddy" is **Theodore Roosevelt**. 
He earned this nickname due to an incident that occurred during a hunting trip in 1902. While on a hunting expedition in Mississippi, Roosevelt's party tracked down a bear, but when Roosevelt reached the location, he found the bear tied to a tree. Feeling it was unsportsmanlike to shoot the defenceless bear, Roosevelt refused to shoot it and ordered it to be set free.
This incident was then depicted in a widely circulated political cartoon by Clifford Berryman, which portrayed Roosevelt sparing the bear. The cartoon caught the attention of the public, and the nickname "**Teddy**" was coined, as a diminutive form of Theodore. The term "teddy bear" also emerged from this incident, as a stuffed toy inspired by the event became popular, with the name given to it as a nod to President Roosevelt's compassionate act.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/803bed22-b379-498c-a93e-9f5485979736)
